src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -21,31 +18,62 @@ import LegendEditor from '../../../plots-and-tables/components/LegendEditor';
 import LabelsDesign from '../../../plots-and-tables/components/LabelsDesign';
 import ColourInversion from '../../../plots-and-tables/components/ColourInversion';
 
-const defaultPlotStylingConfig = {
-  legendEnabled: 'true',
-  legendPosition: 'top',
-  labelsEnabled: true,
-  labelSize: 28,
-  xAxisText: 'Principal Components',
-  yAxisText: 'Proportion of Variance Explained',
-  xDefaultTitle: 'Principal Components',
-  yDefaultTitle: 'Proportion of Variance Explained',
-  titleSize: 12,
-  titleText: 'Scree plot',
-  titleDx: 10,
-  titleAnchor: 'start',
-  masterFont: 'sans-serif',
-  masterSize: 13,
-  xaxisText: 'Principal Components',
-  yaxisText: 'Proportion of Variance Explained',
-  axisTitlesize: 13,
-  axisTicks: 13,
-  axisOffset: 0,
-  transGrid: 10,
-  width: 530,
-  height: 400,
-  maxWidth: 720,
-  maxHeight: 530,
+import isBrowser from '../../../../../../utils/environment';
+
+import loadCellSets from '../../../../../../redux/actions/cellSets/loadCellSets';
+
+// TODO This loadPlotConfig should probably be changed on the redux ticket? because it fetches in a url that seems dedicated to plots and tables:
```

Comment:
is this for the styling settings?

Reply:
Yes

Reply:
Im going to remove this comment and add it to the ticket.